zenodo/config.py

```diff
@@ -585,6 +584,7 @@ def _(x):
 SECURITY_PASSWORD_SALT = "CHANGE_ME"
 SECURITY_REMEMBER_SALT = "CHANGE_ME"
 SECURITY_RESET_SALT = "CHANGE_ME"
+SECURITY_PASSWORD_SCHEMES = ['plaintext', 'pbkdf2_sha512']
```

Comment:
@lnielsen 
passlib asks me to configure it (see https://travis-ci.org/zenodo/zenodo/jobs/131363196 )
If not, it raise the exception `ValueError: hash could not be identified`.
How can I deal with it?